lib/identifiers/pubmed_id.rb

```diff
@@ -1,7 +1,11 @@
 module Identifiers
   class PubmedId
+    ZERO_PADDED_NUMBER = %r{(?<=^|[[:space:]])0*(?!0)(\d+)(?=$|[[:space:]])}
+    PUBMED_URL = %r{https?://(?:www\.)?ncbi\.nlm\.nih\.gov/pubmed/0*(\d+)}i
```

Comment:
Should we also support mobile PubMed URLs, e.g. https://www.ncbi.nlm.nih.gov/m/pubmed/22497409?